Amend 2111.1 release notes with known issue. (#1272)
o3de/o3de.org: content/docs/release-notes/archive/21-11-release-notes.md

  To work around this issue, run AssetProcessor to process project assets before launching Editor for the first time. After all assets are processed initially, Editor can be launched without manually running AssetProcessor.

> * If the user has previously used O3DE and an `o3de_manifest.json` file exists in their `/Users/(username)/.o3de` directory, when they go to build a project using the current installer version O3DE, it may fail with an error similar to the following:

Reviewer: ```suggestion
* If the user has previously used O3DE and an `o3de_manifest.json` file exists in their `/Users/(username)/.o3de` directory, when they go to build a project using the current installer version of O3DE, it may fail with an error similar to the following:
```

> * If the user has previously used O3DE and an `o3de_manifest.json` file exists in their `/Users/(username)/.o3de` directory, when they go to build a project using the current installer version of O3DE, it may fail with an error similar to the following:

Reviewer 2: Assuming this applies to Windows OR Linux, I suggest including the path for each. Also, "<replaceable-path>" is our standard formatting for placeholder directory names.
```suggestion
* If the user has previously used O3DE and an `o3de_manifest.json` file exists in their `<user>/.o3de` (Windows) or `$HOME/.o3de` (Linux) directory, when they go to build a project using the current installer version of O3DE, it may fail with an error similar to the following:
```

  ```cmd
  CMake Error at CMakeLists.txt:10 (find_package):
  By not providing "Findo3de.cmake" in CMAKE_MODULE_PATH this project has
  asked CMake to find a package configuration file provided by "o3de", but
  CMake did not find one.
  Could not find a package configuration file provided by "o3de" with any of
  the following names:
  o3deConfig.cmake
  o3de-config.cmake
  ```

  To work around this issue, perform the following steps:

>     1. Go to the `Users/(username)/.o3de directory (this directory is hidden on some platforms by default) and delete the existing `o3de_manifest.json` file.

Reviewer 2: ```suggestion
    1. Go to the `<user>/.o3de` or `$HOME/.o3de` directory (this directory is hidden on some platforms by default) and delete the existing `o3de_manifest.json` file.
```

>     1. Go to the `Users/(username)/.o3de directory (this directory is hidden on some platforms by default) and delete the existing o3de_manifest.json file
>     1. Launch O3DE again to create a new one
>     1. Attempt the build again

Reviewer: ```suggestion
    1. Go to the `Users/(username)/.o3de directory (this directory is hidden on some platforms by default) and delete the existing `o3de_manifest.json` file.
    1. Launch O3DE again to create a new one.
    1. Attempt the build again.
```

>   At this point the build should succeed since the only reference is to the currently installed version of O3DE

Reviewer: ```suggestion
  At this point the build should succeed since the only reference is to the currently installed version of O3DE.
```
